src/selectionHandler.ts

```diff
@@ -92,7 +108,17 @@ function resolveDetailOf(completion: any): string {
 
 async function handleImports(editor: TextEditor, completion: any) {
     let selection = editor.selection;
-    let completionSelection = new Selection(selection.active.translate(0, -completion.length), selection.active);
+    let lineDelta = 0;
+    let characterDelta = -completion.length;
+
+    if (completion.indexOf(endOfLine) > 0) {
```

Comment:
you already are passing the position to selectionHandler so you can create a Selection by passing it the position and editor.selection.anchor, this code is responsible for the auto imports, so please check that it still works